pkg/ddl/notifier/subscribe.go

```diff
@@ -189,10 +212,18 @@ func (n *ddlNotifier) processEventForHandler(
 		}
 	}()
 
-	// TODO: Should we attach a timeout to this ctx?
+	now := time.Now()
 	if err = handler(ctx, n.ownedSCtx, change.event); err != nil {
 		return errors.Trace(err)
 	}
+	if time.Since(now) > logSlowProcess {
+		logutil.Logger(ctx).Warn("Slow process event",
```

Comment:
I think it's better to have a label like `[ddl_notifier]` for all logs in this module.